docker-compose.prod.yaml

```diff
@@ -5,7 +5,10 @@ services:
   app:
     build: ./app
     ports:
-      - "80:5000"
+      - "443:443"
+    volumes:
+      - /etc/letsencrypt/live/usedgoodreads.com/fullchain.pem:/etc/letsencrypt/live/usedgoodreads.com/fullchain.pem
+      - /etc/letsencrypt/live/usedgoodreads.com/privkey.pem:/etc/letsencrypt/live/usedgoodreads.com/privkey.pem
```

Comment:
It would be great to have docs on how to set letsencrypt up, and what we require on the server.